chain/chunks/src/lib.rs

```diff
@@ -870,6 +898,7 @@ impl ShardsManager {
                 .parts
                 .iter()
                 .filter_map(|(part_ord, part_entry)| {
+                    // Should we include sampled parts here too??
```

Comment:
No, here we only keep parts that someone can request from us in the future, knowing that we own them. So no need to save sampled parts here.